doctordroid/manifest.json

```diff
@@ -0,0 +1,77 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "5e75658c-065e-460f-b9f8-42bf100e361d",
```

Comment:
No action needed: I confirmed this app_uuid matches the `app_uuid` of oauth client ✅